easybuild/easyconfigs/t/TensorFlow/TensorFlow-1.12.0-fosscuda-2018b-Python-2.7.15.eb

```diff
@@ -0,0 +1,108 @@
+    }),
+    ('absl-py', '0.6.1', {
+        'modulename': 'absl',
+        'source_urls': ['https://pypi.python.org/packages/source/a/absl-py'],
+        'checksums': ['87519e3b91a3d573664c6e2ee33df582bb68dca6642ae3cf3a4361b1c0a4e9d6'],
+    }),
+    ('astor', '0.7.1', {
+        'source_urls': ['https://pypi.python.org/packages/source/a/astor'],
+        'checksums': ['95c30d87a6c2cf89aa628b87398466840f0ad8652f88eb173125a6df8533fb8d'],
+    }),
+    ('gast', '0.2.0', {
+        'source_urls': ['https://pypi.python.org/packages/source/g/gast'],
+        'checksums': ['7068908321ecd2774f145193c4b34a11305bd104b4551b09273dfd1d6a374930'],
+    }),
+    ('grpcio', '1.16.0', {
+        'modulename': 'grpc',
+        'source_urls': ['https://pypi.python.org/packages/source/g/grpcio'],
+        'checksums': ['0cc5f2d3ee21c642d8982f197c83053fd3a8cbcd6a60240d8c87c6c256b10d57'],
+    }),
+    ('Markdown', '3.0.1', {
+        'source_urls': ['https://pypi.python.org/packages/source/M/Markdown'],
+        'checksums': ['d02e0f9b04c500cde6637c11ad7c72671f359b87b9fe924b2383649d8841db7c'],
+    }),
+    ('tensorboard', version, {
+        'source_tmpl': 'tensorboard-%(version)s-py2-none-any.whl',
+        'source_urls': ['https://pypi.python.org/packages/source/t/tensorboard'],
+        'unpack_sources': False,
+        'checksums': ['537603db949e10d2f5f201d88b073f3f8fb4e4c311d5541e1d4518aa59aa8daa'],
+    }),
+    ('termcolor', '1.1.0', {
+        'source_urls': ['https://pypi.python.org/packages/source/t/termcolor'],
+        'checksums': ['1d6d69ce66211143803fbc56652b41d73b4a400a2891d7bf7a1cdf4c02de613b'],
+    }),
+    ('Werkzeug', '0.14.1', {
+        'source_urls': ['https://pypi.python.org/packages/source/W/Werkzeug'],
+        'checksums': ['c3fd7a7d41976d9f44db327260e263132466836cef6f91512889ed60ad26557c'],
+    }),
+    ('Keras-Applications', '1.0.6', {
+        'modulename': 'keras_applications',
+        'source_tmpl': 'Keras_Applications-%(version)s.tar.gz',
+        'source_urls': ['https://pypi.python.org/packages/source/K/Keras-Applications'],
+        'checksums': ['a03af60ddc9c5afdae4d5c9a8dd4ca857550e0b793733a5072e0725829b87017'],
+    }),
+    ('Keras-Preprocessing', '1.0.5', {
+        'modulename': 'keras_preprocessing',
+        'source_tmpl': 'Keras_Preprocessing-%(version)s.tar.gz',
+        'source_urls': ['https://pypi.python.org/packages/source/K/Keras-Preprocessing'],
+        'checksums': ['ef2e482c4336fcf7180244d06f4374939099daa3183816e82aee7755af35b754'],
+    }),
+    (name, version, {
+        'patches': [
+            'TensorFlow-1.11.0_swig-env.patch',
+            'TensorFlow-1.11.0_remove-msse-hardcoding.patch',
+            'TensorFlow-1.12.0_lrt-flag.patch',
+            'TensorFlow-1.12.0_dont_expand_cuda_cudnn_path.patch'
+        ],
+        'source_tmpl': 'v%(version)s.tar.gz',
+        'source_urls': ['https://github.com/tensorflow/tensorflow/archive/'],
```

Comment:
also messed up order here: should be `source_urls`, `source_tmpl`, `patches`, `checksums`